edb/server/compiler/errormech.py

```diff
@@ -230,10 +248,10 @@ def interpret_backend_error(schema, fields):
             message, details=detail if detail else None)
 
     elif code == PGError.InvalidTextRepresentation:
```

Comment:
Make a frozenset of codes that require schema. Have just one `if code in SCHEMA_CODES` clause.

Reply:
I've put the codes that unambiguously require schema in `SCHEMA_CODES`. There are some error codes (notably, constraint_errors) that don't always require the schema and which I didn't put into the `SCHEMA_CODES` set.